templates/index.html

```diff
@@ -35,4 +35,28 @@ <h2 class="p-heading--three"><a href="{{ featured_post.relative_link }}">{{ feat
 
   {% include "posts.html" %}
 
+<div class="p-strip is-shallow is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Webinars</h2>
+    {% for webinar in webinars %}
+      <div class="p-media-object" itemscope itemtype="http://schema.org/Event">
+        <div class="p-media-object__details">
```

Comment:
Can you use the `thumbnail_url` in the RSS to display the media objects image please.

Reply:
Done